phpunit.xml

```diff
@@ -29,5 +29,6 @@
   <php>
     <server name="APP_ENV" value="testing"/>
     <env name="DB_CONNECTION" value="testing"/>
+    <env name="APP_KEY" value="AckfSECXIvnK5r28GVIWUAxmbBSjTsmF"/>
```

Comment:
Could we move this to below APP_ENV just so they're grouped please?

Reply:
Absolutely. I will update

Reply:
Cleaned up :+1: